
  <!DOCTYPE html>
  <html>
    <head>
      <meta name="viewport" content="width=device-width, initial-scale=1" />
      <style>
      highlight {
      color: green;
      }

      illumina_p5 {color:#08519c;}
      illumina_p7 {color:#a50f15;}
      nextera_read1 {color:#bcbddc;}
      nextera_read2 {color:#9ebcda;}
      truseq_read1 {color:#4a1486;}
      truseq_read2 {color:#6a51a3;}
      ME {color:#969696;}
      s5 {color:#6baed6;}
      s7 {color:#fc9272;}
      barcode {color:#f768a1;}
      umi {color:#807dba;}
      gDNA {color:#f03b20;}
    cDNA {color:#7e331f;}
      </style>
    </head>
    <body>
      <div style="width: 75%; margin: 0 auto">
        <h6><a href="../../index.html">Back</a></h6>
        <div id="assay">
          <h1 style="text-align: center">snmCT-seq</h1>
  <ul>
    <li>
      <a href="https://doi.org/10.1016/j.xgen.2022.100107"
        >https://doi.org/10.1016/j.xgen.2022.100107</a
      >
    </li>
    <li>
      Single-nucleus methylcytosine and transcriptome sequencing (snmCT-seq). Specification assumes 2x151bp paired-end reads with 10bp dual indices based on snmC-seq3 and SMART-seq2. Bisulfite-converted gDNA reads (methyl) and cDNA reads (RNA) are separated in silico.
    </li>
    <li>methyl, rna</li>
  </ul>
    
        </div>
        <div id="assay_spec">
          <h2>Final library</h2>
          
  <h6 style="text-align: center">methyl</h6>
  <pre
    style="overflow-x: auto; text-align: left; background-color: #f6f8fa"
  >
<illumina_p5>AATGATACGGCGACCACCGAGATCTACAC</illumina_p5><index5>NNNNNNNNNNN</index5><truseq_read1>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</truseq_read1><barcode>NNNNNNNN</barcode><methyl>HHHHHHHHH</methyl><methyl>X</methyl><methyl>X</methyl><linker>Y</linker><truseq_read2>AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC</truseq_read2><barcode>NNNNNNNNNNNN</barcode><illumina_p7>ATCTCGTATGCCGTCTTCTGCTTG</illumina_p7></pre>
    
<ol><li>
    <details>
    <summary>illumina_p5</summary>
    <ul>
      <li>region_type: illumina_p5</li>
      <li>sequence_type: fixed</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<illumina_p5>AATGATACGGCGACCACCGAGATCTACAC</illumina_p5></pre
        >
      </li>
      <li>min_len: 29</li>
      <li>max_len: 29</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li><li><details>
    <summary>I2.fastq.gz</summary>
    <ul>
      <li>region_type: fastq</li>
      <li>sequence_type: joined</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<index5>NNNNNNNNNNN</index5></pre
        >
      </li>
      <li>min_len: 10</li>
      <li>max_len: 10</li>
      <li>onlist: None</li>
      <li> regions: <ol><li><details>
    <summary>i5_barcode</summary>
    <ul>
      <li>region_type: index5</li>
      <li>sequence_type: onlist</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<index5>NNNNNNNNNNN</index5></pre
        >
      </li>
      <li>min_len: 10</li>
      <li>max_len: 10</li>
      <li>onlist: i5_onlist.txt (md5: c36c18a6ee1603758767878b794335f1)</li>
      <li> regions: 
      </li>
  </details>
    </li></ol>
      </li>
  </details>
    </li><li><details>
    <summary>truseq_read1</summary>
    <ul>
      <li>region_type: truseq_read1</li>
      <li>sequence_type: fixed</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<truseq_read1>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</truseq_read1></pre
        >
      </li>
      <li>min_len: 33</li>
      <li>max_len: 33</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li><li><details>
    <summary>R1.fastq.gz</summary>
    <ul>
      <li>region_type: fastq</li>
      <li>sequence_type: joined</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<barcode>NNNNNNNN</barcode><methyl>HHHHHHHHH</methyl><methyl>X</methyl></pre
        >
      </li>
      <li>min_len: 151</li>
      <li>max_len: 151</li>
      <li>onlist: None</li>
      <li> regions: <ol><li><details>
    <summary>cellbarcode</summary>
    <ul>
      <li>region_type: barcode</li>
      <li>sequence_type: onlist</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<barcode>NNNNNNNN</barcode></pre
        >
      </li>
      <li>min_len: 8</li>
      <li>max_len: 8</li>
      <li>onlist: cb_onlist.txt (md5: 0691bed03feb775d9aff8b37fb4dc94b)</li>
      <li> regions: 
      </li>
  </details>
    </li><li><details>
    <summary>primer9H</summary>
    <ul>
      <li>region_type: methyl</li>
      <li>sequence_type: random</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<methyl>HHHHHHHHH</methyl></pre
        >
      </li>
      <li>min_len: 9</li>
      <li>max_len: 9</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li><li><details>
    <summary>bisulfite_gDNA-1</summary>
    <ul>
      <li>region_type: methyl</li>
      <li>sequence_type: random</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<methyl>X</methyl></pre
        >
      </li>
      <li>min_len: 1</li>
      <li>max_len: 134</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li></ol>
      </li>
  </details>
    </li><li><details>
    <summary>R2.fastq.gz</summary>
    <ul>
      <li>region_type: fastq</li>
      <li>sequence_type: joined</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<methyl>X</methyl><linker>Y</linker></pre
        >
      </li>
      <li>min_len: 151</li>
      <li>max_len: 151</li>
      <li>onlist: None</li>
      <li> regions: <ol><li><details>
    <summary>bisulfite_gDNA-2</summary>
    <ul>
      <li>region_type: methyl</li>
      <li>sequence_type: random</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<methyl>X</methyl></pre
        >
      </li>
      <li>min_len: 1</li>
      <li>max_len: 150</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li><li><details>
    <summary>adaptase_CTs</summary>
    <ul>
      <li>region_type: linker</li>
      <li>sequence_type: fixed</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<linker>Y</linker></pre
        >
      </li>
      <li>min_len: 1</li>
      <li>max_len: 15</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li></ol>
      </li>
  </details>
    </li><li><details>
    <summary>truseq_read2</summary>
    <ul>
      <li>region_type: truseq_read2</li>
      <li>sequence_type: fixed</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<truseq_read2>AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC</truseq_read2></pre
        >
      </li>
      <li>min_len: 34</li>
      <li>max_len: 34</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li><li><details>
    <summary>I1.fastq.gz</summary>
    <ul>
      <li>region_type: fastq</li>
      <li>sequence_type: joined</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<barcode>NNNNNNNNNNNN</barcode></pre
        >
      </li>
      <li>min_len: 10</li>
      <li>max_len: 10</li>
      <li>onlist: None</li>
      <li> regions: <ol><li><details>
    <summary>i7_barcode</summary>
    <ul>
      <li>region_type: barcode</li>
      <li>sequence_type: onlist</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<barcode>NNNNNNNNNNNN</barcode></pre
        >
      </li>
      <li>min_len: 10</li>
      <li>max_len: 10</li>
      <li>onlist: i7_onlist.txt (md5: 47874d94562f3b00125385a6c60daf9d)</li>
      <li> regions: 
      </li>
  </details>
    </li></ol>
      </li>
  </details>
    </li><li><details>
    <summary>illumina_p7</summary>
    <ul>
      <li>region_type: illumina_p7</li>
      <li>sequence_type: fixed</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<illumina_p7>ATCTCGTATGCCGTCTTCTGCTTG</illumina_p7></pre
        >
      </li>
      <li>min_len: 24</li>
      <li>max_len: 24</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    
    </li></ol>
  <h6 style="text-align: center">rna</h6>
  <pre
    style="overflow-x: auto; text-align: left; background-color: #f6f8fa"
  >
<illumina_p5>AATGATACGGCGACCACCGAGATCTACAC</illumina_p5><index5>NNNNNNNNNNN</index5><truseq_read1>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</truseq_read1><barcode>NNNNNNNN</barcode><rna>HHHHHHHHH</rna><rna>X</rna><rna>X</rna><linker>Y</linker><truseq_read2>AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC</truseq_read2><barcode>NNNNNNNNNNNN</barcode><illumina_p7>ATCTCGTATGCCGTCTTCTGCTTG</illumina_p7></pre>
    
<ol><li>
    <details>
    <summary>illumina_p5</summary>
    <ul>
      <li>region_type: illumina_p5</li>
      <li>sequence_type: fixed</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<illumina_p5>AATGATACGGCGACCACCGAGATCTACAC</illumina_p5></pre
        >
      </li>
      <li>min_len: 29</li>
      <li>max_len: 29</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li><li><details>
    <summary>I2.fastq.gz</summary>
    <ul>
      <li>region_type: fastq</li>
      <li>sequence_type: joined</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<index5>NNNNNNNNNNN</index5></pre
        >
      </li>
      <li>min_len: 10</li>
      <li>max_len: 10</li>
      <li>onlist: None</li>
      <li> regions: <ol><li><details>
    <summary>i5_barcode</summary>
    <ul>
      <li>region_type: index5</li>
      <li>sequence_type: onlist</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<index5>NNNNNNNNNNN</index5></pre
        >
      </li>
      <li>min_len: 10</li>
      <li>max_len: 10</li>
      <li>onlist: i5_onlist.txt (md5: c36c18a6ee1603758767878b794335f1)</li>
      <li> regions: 
      </li>
  </details>
    </li></ol>
      </li>
  </details>
    </li><li><details>
    <summary>truseq_read1</summary>
    <ul>
      <li>region_type: truseq_read1</li>
      <li>sequence_type: fixed</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<truseq_read1>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</truseq_read1></pre
        >
      </li>
      <li>min_len: 33</li>
      <li>max_len: 33</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li><li><details>
    <summary>R1.fastq.gz</summary>
    <ul>
      <li>region_type: fastq</li>
      <li>sequence_type: joined</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<barcode>NNNNNNNN</barcode><rna>HHHHHHHHH</rna><rna>X</rna></pre
        >
      </li>
      <li>min_len: 151</li>
      <li>max_len: 151</li>
      <li>onlist: None</li>
      <li> regions: <ol><li><details>
    <summary>cellbarcode</summary>
    <ul>
      <li>region_type: barcode</li>
      <li>sequence_type: onlist</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<barcode>NNNNNNNN</barcode></pre
        >
      </li>
      <li>min_len: 8</li>
      <li>max_len: 8</li>
      <li>onlist: cb_onlist.txt (md5: 0691bed03feb775d9aff8b37fb4dc94b)</li>
      <li> regions: 
      </li>
  </details>
    </li><li><details>
    <summary>primer9H</summary>
    <ul>
      <li>region_type: rna</li>
      <li>sequence_type: random</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<rna>HHHHHHHHH</rna></pre
        >
      </li>
      <li>min_len: 9</li>
      <li>max_len: 9</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li><li><details>
    <summary>transcriptome-1</summary>
    <ul>
      <li>region_type: rna</li>
      <li>sequence_type: random</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<rna>X</rna></pre
        >
      </li>
      <li>min_len: 1</li>
      <li>max_len: 134</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li></ol>
      </li>
  </details>
    </li><li><details>
    <summary>R2.fastq.gz</summary>
    <ul>
      <li>region_type: fastq</li>
      <li>sequence_type: joined</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<rna>X</rna><linker>Y</linker></pre
        >
      </li>
      <li>min_len: 151</li>
      <li>max_len: 151</li>
      <li>onlist: None</li>
      <li> regions: <ol><li><details>
    <summary>transcriptome-2</summary>
    <ul>
      <li>region_type: rna</li>
      <li>sequence_type: random</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<rna>X</rna></pre
        >
      </li>
      <li>min_len: 1</li>
      <li>max_len: 150</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li><li><details>
    <summary>adaptase_CTs</summary>
    <ul>
      <li>region_type: linker</li>
      <li>sequence_type: fixed</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<linker>Y</linker></pre
        >
      </li>
      <li>min_len: 1</li>
      <li>max_len: 15</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li></ol>
      </li>
  </details>
    </li><li><details>
    <summary>truseq_read2</summary>
    <ul>
      <li>region_type: truseq_read2</li>
      <li>sequence_type: fixed</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<truseq_read2>AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC</truseq_read2></pre
        >
      </li>
      <li>min_len: 34</li>
      <li>max_len: 34</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    </li><li><details>
    <summary>I1.fastq.gz</summary>
    <ul>
      <li>region_type: fastq</li>
      <li>sequence_type: joined</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<barcode>NNNNNNNNNNNN</barcode></pre
        >
      </li>
      <li>min_len: 10</li>
      <li>max_len: 10</li>
      <li>onlist: None</li>
      <li> regions: <ol><li><details>
    <summary>i7_barcode</summary>
    <ul>
      <li>region_type: barcode</li>
      <li>sequence_type: onlist</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<barcode>NNNNNNNNNNNN</barcode></pre
        >
      </li>
      <li>min_len: 10</li>
      <li>max_len: 10</li>
      <li>onlist: i7_onlist.txt (md5: 47874d94562f3b00125385a6c60daf9d)</li>
      <li> regions: 
      </li>
  </details>
    </li></ol>
      </li>
  </details>
    </li><li><details>
    <summary>illumina_p7</summary>
    <ul>
      <li>region_type: illumina_p7</li>
      <li>sequence_type: fixed</li>
      <li>
        sequence:
        <pre
        style="
        overflow-x: auto;
        text-align: left;
        margin: 0;
        display: inline;
        "
        >
<illumina_p7>ATCTCGTATGCCGTCTTCTGCTTG</illumina_p7></pre
        >
      </li>
      <li>min_len: 24</li>
      <li>max_len: 24</li>
      <li>onlist: None</li>
      <li> regions: 
      </li>
  </details>
    
    </li></ol>
        </div>
      </div>
    </body>
  </html>
    
